### ISSUE TITLE
Cannot add new entries in Tree view - Flutter Flow

### ISSUE THREAD

**Issue by looganxx** [SanjaySodani/json_editor_flutter#6]
return Container(
        width: widget.width,
        height: widget.height,
        child: JsonEditor(
          onChanged: (_) {},
          json: jsonEncode(widget.json),
          themeColor: Color(0xFF4B39EF),
        ));
        
this is my configuration

I'm able to edit the already existing values of json object, but i can't add new string, list or object.
Instead, in text visualization I can make any modification.

What am I missing?

**Comment by SanjaySodani** [comment]
@looganxx Have you tried the more options(3 dots) on the left side?

**Comment by looganxx** [comment]
@SanjaySodani yes, of course. I used the three dots, but when I click on String, List or others options this make no effect to the JSON content.

**Comment by SanjaySodani** [comment]
@looganxx I gave it a try and it works for me. Even on the demo app (from the package landing page). 

And was the object expanded too?

**Comment by looganxx** [comment]
yes, the object expands. Even for me, the demo app works. I just wanted to know if I am doing something wrong in my implementation, because as I said, in text mode I can edit.

**Comment by SanjaySodani** [comment]
No, you are not missing anything.

**Comment by looganxx** [comment]
I am using FlutterFlow for the implementation, maybe something with it does not work well? Do you have the opportunity to test your package on FF?

**Comment by SanjaySodani** [comment]
No, I have never. Let me try it out. I will revert, it might take some time as I'm new to FF.
